components/page-header/index.jsx

```diff
@@ -116,11 +130,18 @@ const defaultProps = {
  * The PageHeader component adds PageHeader, PageHeader.Info, PageHeader.Title, PageHeader.DetailRow, and PageHeader.DetailBlock.
  */
 class PageHeader extends Component {
+	constructor(props) {
+		super(props);
+		checkProps(PAGE_HEADER, props, componentDoc);
```

Comment:
`checkProps` should in `componentDidMount`, so that it only fires off it it mounts to the DOM.

Reply:
Modified.